### ISSUE TITLE
Docker repository doesn't exist

### ISSUE THREAD

**Issue by alexanch** [nshaud/DeepHyperX#2]
Hi, thank you very much for such a great tool. 
I was trying like to grab a Docker image but seems that it doesn't exist. Would you please upload it once again so I could copy the image?

(base) alexanch@alexanch:~/docker$ docker pull registry.gitlab.inria.fr/naudeber/deephyperx:preview
Error response from daemon: manifest for registry.gitlab.inria.fr/naudeber/deephyperx:preview not found

**Comment by nshaud** [comment]
Yes, the Docker registry at Inria has been closed. I haven't got the chance yet to migrate to Docker hub but you can still build the container locally with `docker build`.
